homeassistant/components/synology_dsm/config_flow.py

```diff
@@ -27,46 +31,49 @@
 )
 from .const import DOMAIN  # pylint: disable=unused-import
 
+_LOGGER = logging.getLogger(__name__)
+
+
+def _schema_with_defaults(user_input=None):
+    if user_input is None:
+        user_input = {}
+
+    return vol.Schema(
+        {
+            vol.Optional(
+                CONF_NAME, default=user_input.get(CONF_NAME, DEFAULT_NAME)
+            ): str,
```

Comment:
Changing existing code means I see it again… anyone know why we can configure the name? And can't we query any kind of name from the device ?

Reply:
If you have two identical Syno boxes (I do for live backups).  They come out of the box named "Synology DS918+" so hard to tell which one is which.  Going to hide it when they do the discovery path

Reply:
Name removed